src/main.py

```diff
@@ -27,13 +30,15 @@
 print(f"{len(new_podcasts)} new podcasts found...")
 
 last_youtube_publication_date = get_last_youtube_publication_date(latest_video['id']) # -> "The last youtube publication date"
+
+extra_day=0
 for podcast in new_podcasts:
```

Comment:
extra_day devrait être au pluriel et tu peux économiser des lignes en incluant l'index dans ta boucle for